sql/15.0/type.sql

```diff
@@ -266,5 +266,163 @@ SELECT * FROM "type_DOUBLE"; -- OK
 --Testcase 107:
 ALTER FOREIGN TABLE "type_DOUBLE" ALTER COLUMN col TYPE float8;
 
+--Testcase 108:
+DROP FOREIGN TABLE "type_UUID";
+--Testcase 109:
+CREATE FOREIGN TABLE "type_UUID"( "i" int OPTIONS (key 'true'), "u" uuid) SERVER sqlite_svr OPTIONS (table 'type_UUID');
+--Testcase 110:
+ALTER FOREIGN TABLE "type_UUID" ALTER COLUMN "u" TYPE text;
+--Testcase 111:
+INSERT INTO "type_UUID" ("i", "u") VALUES (1, 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11');
```

Comment:
Could you add explain case to show deparsed SQL query?    
e.g.
```sql
EXPLAIN VERBOSE
INSERT INTO "type_UUID" ("i", "u") VALUES (1, 'a0eebc99-9c0b-4ef8-bb6d-6bb9bd380a11');
```

Reply:
This is not really test case, because for TC 111 FDW table column data type is `text`, not `uuid`. This is data filling. I have added `EXPLAIN VERBOSE` to some real DML where FDW table column data type is `uuid`. Please, verify.
_Note_: `EXPLAIN VERBOSE` for `INSERT` doesn't show SQLite query, but for `UPDATE` or `DELETE` or `SELECT` does.